kore/src/SimpleSMT.hs

```diff
@@ -944,6 +1006,12 @@ store :: SExpr {- ^ array -}     ->
          SExpr
 store x y z = fun "store" [x,y,z]
 
+-- | Quantifiers: forall
```

Comment:
Did you change the default theory in `SMT.defaultConfig` to allow quantifiers?

Reply:
No, but for some reason it seems to work. I have no clue why. I was reluctant at first to add this, but, since I was only using it in a test, I thought it should be fine.